docs/core/get-started.md

```diff
@@ -79,4 +79,4 @@ You can get started developing .NET Core application by following these step-by-
 
 See the [Prerequisites for macOS development](macos-prerequisites.md) article for a list of the supported OS X / macOS versions.
 
-***
\ No newline at end of file
+***
```

Comment:
what is this?

Reply:
Looks like it was there before. Not sure why.

Reply:
This is part of the tab syntax @Thraka